src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -662,6 +668,7 @@ public static SnowflakeResultSetSerializableV1 create(
     resultSetSerializable.snowflakeConnectionString = sfSession.getSnowflakeConnectionString();
     resultSetSerializable.networkTimeoutInMilli = sfSession.getNetworkTimeoutInMilli();
     resultSetSerializable.authTimeout = 0;
+    resultSetSerializable.maxHttpRetries = sfSession.getMaxHttpRetries();
```

Comment:
Can we allow as configurable parameter?

Reply:
yes, this comes from connection parameter.